alfresco-transform-misc/alfresco-transform-misc/src/main/java/org/alfresco/transformer/transformers/TextToPdfContentTransformer.java

```diff
@@ -207,7 +333,7 @@ public PDDocument createPDFFromText(Reader text, int pageLimit)
                 //calculate font height and increase by 5 percent.
                 height = height * getFontSize() * 1.05f;
                 doc = new PDDocument();
-                BufferedReader data = new BufferedReader(text);
+                BufferedReader data = (text instanceof BufferedReader) ? (BufferedReader) text : new BufferedReader(text);
                 String nextLine;

```

Comment:
It is always a BufferedReader, but I did not want to change the method signature. Makes debugging simpler without nested Readers.